### PR TITLE
Team flexbox

### LINKED ISSUE #8: Research how to make the page look like the wireframe
Figure out how to format the page to have a tall vertical div on the left and stacked ones on the right

### LINKED ISSUE #9: Create elements within HTML files and add ID tags
div id="profileDiv"
nav id= "navbar"
div id= "mainContent"
div id= "formDiv"
footer id= "footer"

### PR DESCRIPTION
**Pull request by NathanBlackman**
Created all the containers and divs, assigned them id's and classes, styled/flexboxed them, and updated README with the deploy link.

Closes #9 

Closes #8

### CODE REVIEW

File: content.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.profile-div{
+  background-color: blue;
+  width: 445px;
+  height: 1468px;
+}
+.profile-div:before {
```
Comment (NathanBlackman) [inline content.css:6]: This CSS file is for visualizing the page. We will eventually delete this file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (TrinityChristiana) [inline index.html:1]: There's a console log in here!

Comment (NathanBlackman) [reply index.html:1]: no...

**Review by austincparker — APPROVED** [review]
it's beautiful

**Review by pmpurcell — APPROVED** [review]
Beautiful!

**Pull request merged**